src/app/tables/artifacts-table/artifacts-table.component.html

```diff
@@ -1,9 +1,11 @@
 <app-table
     [title]="title"
+    [csvFileName]="csvFileName"
     [items]="formattedArtifacts"
     [preformattedColumns]="preformattedColumns"
     [defaultColumns]="defaultColumns"
     [uidColumn]="uidColumn"
     [linkColumnMappings]="linkColumnMappings"
     [defaultSortedColumn]="defaultSortedColumn"
+    [tableSpacing]="tableSpacing"
```

Comment:
Since the `tableSpacing` variable doesn't ever change, it doesn't need to be a variable. It'd be simpler if you just did `tableSpacing="false"`